EasyPost/Pickup.cs

```diff
@@ -33,14 +34,32 @@ public class Pickup : Resource
         [JsonProperty("name")]
         public string name { get; set; }
         [JsonProperty("pickup_rates")]
-        public List<Rate> pickup_rates { get; set; }
+        public List<PickupRate> pickup_rates { get; set; }
         [JsonProperty("reference")]
         public string reference { get; set; }
         [JsonProperty("status")]
         public string status { get; set; }
         [JsonProperty("updated_at")]
         public DateTime? updated_at { get; set; }
 
+        /// <summary>
+        ///     Get the pickup rates as a list of Rate objects.
+        /// </summary>
+        /// <returns>List of Rate objects.</returns>
+        internal List<Rate> Rates
+        {
+            get
+            {
+                List<Rate> rates = new List<Rate>();
+                foreach (PickupRate pickupRate in pickup_rates)
+                {
+                    rates.Add(pickupRate);
```

Comment:
Just a question, where do you convert the `pickupRate` to `rate`?

Reply:
`PickupRate` is a subclass of `Rate`, so I can put a `PickupRate` object into a list of `Rate` objects without needing to cast.